vendor-bin/php-codesniffer/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "require": {
+        "php": "^8.0",
```

Comment:
Might be worth adding version 7.4 in these composer.json files to match the main composer file